framework/src/st_things/things_stack/things_stack.c

```diff
@@ -173,8 +177,63 @@ int things_register_easysetup_state_func(things_get_easysetup_state_func_type fu
 	return res;
 }
 
+#ifdef CONFIG_NETUTILS_NTPCLIENT
+static void ntp_link_error(void)
```

Comment:
if this function is used only inside of things_stack, rather declear as a static,
otherwise, how about get parameters and pass to waitng time limits on
 else if ((init_tp.tv_sec + 10) < sync_tp.tv_sec)
?

Reply:
Yes, I applied